index.js

```diff
@@ -57,6 +58,8 @@ sequencer.setUI({
 sequencer.loadImages(program.image,function(){
     console.warn('\x1b[33m%s\x1b[0m', "The execution will be async\nYou may not see the output for a few seconds or minutes")
 
+    if(program.basic) console.log("Basic mode is enabled")
```

Comment:
Can we add `, outputting only final image` and also on line 19 in the `-h` help command? Thanks, this is great!